### ISSUE TITLE
Accessibility issues regarding DTP for blind (screen reader) users 

### ISSUE THREAD

**Issue by hamidrezaabroshan** [eclipse-jdt/.github#18]
There are some accessibility issues regarding DTP for blind users
utilize DTP by screen readers.

0. While navigating on the cells of a table, by pressing the up or
down or left or right arrow keys, cell value is not reported by the
screen-reader. cell values should be reported by the screen-reader but
with applying following rules

1.When navigating between the columns by left or right arrow keys, the
column's  headers and column's number are not reported.
when navigating between the columns, the screen-reader should say
column number, column's header, cell value.

2. when navigating between the rows by up and down keys, the row
number is not reported.
when navigating between the rows, the screen-reader should say row
number, row name, cell value.

note:
1. when navigating up and down, on a column,, it is not needed to
report the column numbers.
2. When navigating left and right, on a row, it is not needed to
report the row numbers.

3. When I execute a query using SQL Scrapbook, each row of the the
result table is read by the screen-reader, similar a semicolon
separated line of a text, .
The result should be a table that a blind user can navigate on it
using arrow keys as the same ordinary tables of the database.
Thanks.

**Comment by fapdash** [comment]
@hamidrezaabroshan Can you please report this to the DTP bug tracker? Link: https://bugs.eclipse.org/bugs/enter_bug.cgi?product=Data%20Tools
You'll need an https://www.eclipse.org/ account to log in to the Eclipse Bugzilla.

**Comment by hamidrezaabroshan** [comment]
reported.
https://bugs.eclipse.org/bugs/show_bug.cgi?id=580965
Thanks.
